include/mull/MutationsFinder.h

```diff
@@ -20,7 +21,9 @@ class Testee;
 class MutationsFinder {
 public:
   explicit MutationsFinder(std::vector<std::unique_ptr<Mutator>> mutators,
-                           const Configuration &config);
+                           const Configuration &config,
+                           const FilePathFilter &filePathFilter,
+                           const InstructionFilter &instructionFilter);
```

Comment:
Could we maybe merge these two filters into one and then do:

`shouldExcludePath` and `shouldExcludeInstruction`?